src/renderer/components/dock/create-resource.tsx

```diff
@@ -21,6 +24,26 @@ interface Props {
 @observer
 export class CreateResource extends React.Component<Props> {
   @observable error = "";
+  @observable templates:GroupSelectOption<SelectOption>[] = [];
+
+  componentDidMount() {
+    createResourceStore.getMergedTemplates().then(v => this.updateGroupSelecOptions(v));
+    createResourceStore.watchUserTemplates(() => createResourceStore.getMergedTemplates().then(v => this.updateGroupSelecOptions(v)));
+  }
+
+  updateGroupSelecOptions(templates :{[x:string]: string[]}) {
```

Comment:
`updateGroupSelecOptions` -> `updateGroupSelectOptions`

Reply:
To add `templates: Record<string, string[]>`